docs/_reporting/hubs/README.md

```diff
@@ -82,13 +95,15 @@ _<sup>2) EA billing scopes can be exported to any tenant today. Simply sign in t
 
 ## 📦 What's included
 
-The FinOps hub template includes the following resources:
+The default deployment of FinOps hub template includes the following resources:
 
 - Storage account (Data Lake Storage Gen2) to hold all cost data.
 - Data Factory instance to manage data ingestion and cleanup.
 - Key Vault to store the Data Factory system managed identity credentials.
 
-Once deployed, you can report on the data in Power BI or by connecting to the storage account directly.
+If you would like to use the new Azure Data Explorer features, you will need to enbale the parameter **dataExplorerName**. To learn more about the parameters needed to deploy Hubs with ADX, refer to the [template](../hubs/template.md) page.
+
```

Comment:
I don't think this is needed here. We're just explaining what's in the template at a high level. We shouldn't get into the details of how to enable features.
```suggestion
```